client/src/components/DataFiles/DataFilesModals/DataFilesManageProjectModal.jsx

```diff
@@ -28,13 +28,21 @@ const DataFilesManageProjectModal = () => {
     };
   });
 
-  const isUserOrGuest = members
+  const canEdit = members
     .filter((member) => member.user.username === user.username)
     .map(
       (currentUser) =>
-        currentUser.access === 'edit' || currentUser.access === 'read'
+        currentUser.access === 'edit' || currentUser.access === 'owner'
```

Comment:
Users with `edit` as their access level have write access to files on the workspace but NOT write access on the Tapis system. On the backend we only allow system owners to edit the system, otherwise it becomes confusing since MODIFY at the system level doesn't allow you to add/remove users. We don't have an "admin" role anymore since functionally it would only let you change the title/description.

I think the change I'd suggest here is to rename the flag to `canEditSystem` and set it to true only if the current user is the system owner.

Reply:
Ahh, thanks for the clarification - I'll implement that suggestion